test/src/e2e_vm_tests/mod.rs

```diff
@@ -233,6 +233,18 @@ fn build_file_checker(content: &str) -> Result<filecheck::Checker, String> {
     Ok(checker.finish())
 }
 
+fn check_file_checker(checker: filecheck::Checker, name: &String, output: &str) {
```

Comment:
could we document this function, since its name is not particularly descriptive?